fission/function.go

```diff
@@ -497,3 +536,43 @@ func fnPods(c *cli.Context) error {
 
 	return err
 }
+
+func fnTest(c *cli.Context) error {
+	fnName := c.String("name")
+	routerURL := os.Getenv("FISSION_ROUTER")
+	if len(routerURL) == 0 {
+		fatal("Need FISSION_ROUTER set to your fission router.")
+	}
+
+	url := fmt.Sprintf("http://%s/fission-function/%s", routerURL, fnName)
+
+	maxRetries := 3
```

Comment:
I don't think this command needs retries, let's keep it simple and the user can retry if they want.